tests/unit/services/pilas-bloques-api-test.js

```diff
@@ -66,9 +66,20 @@ module('Unit | Service | pilas-bloques-api', function (hooks) {
     assert.notOk(api.loading.login)
   })
 
-  test('should add session to body', async function (assert) {
+  test('should add context to body', async function (assert) {
     await api.login({})
-    assert.ok(fetchCallBody().session)
+    const context = fetchCallBody().context
+    assert.ok(context.answers)
+    assert.ok(context.browserId)
+    assert.ok(context.firstInteraction)
+    assert.ok(context.id)
+    assert.ok(context.online)
+    assert.ok(context.userId)
+  })
```

Comment:
Tal vez el test podria tener un mejor nombre? Algo del estilo como 'should add context to body with the intended structure'?

Reply:
![image](https://user-images.githubusercontent.com/4098184/113192992-d76b7d80-9235-11eb-8fef-77ba67fb682c.png)

Me interesa más tener una abstracción para decir: "Este objeto tiene estas properties", que además de ocupar bastante espacio se repite con los otros tests que agregaste.
```
assertHasProps(assert, context, 'answers', 'browserId', ....)
```